hansol/app.js

```diff
@@ -0,0 +1,76 @@
+const http = require('http')
+
+const express = require('express')
+const morgan = require('morgan')
+const cors = require('cors')
+const dotenv = require('dotenv')
+const { DataSource } = require('typeorm')
+dotenv.config()
+
+const app = express();
+const server = http.createServer(app)
+const PORT = process.env.PORT;
+ 
+const dataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME, 
+    password: process.env.TYPEORM_PASSWORD, 
+    database: process.env.TYPEORM_DATABASE, 
+})
+
+dataSource.initialize()
+
+    .then(()=>{
+        console.log('Data Source has been initialized');
+    })
+    .catch((err)=> {
+        console.error('Error during Data Source initialization', err)
+        dataSource.destroy()
+    })
+
+
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('dev'));
+ 
+app.get('/ping', (req,res)=> {
+    res.json({message: 'pong'});
+});
+
+app.get('/users', async (req,res) => {
+    await dataSource.query(
+        `SELECT
+                users.name,
+                users.email,
+                users.password
+        from users`
+
+        ,(err,rows)=>{
+            res.status(200).json(rows);
+        }
+    );    
+});
```

Comment:
한솔님 ! SQL 쿼리문 포매팅을 맞춰서 수정해 주세요 !
```suggestion
app.get('/users', async (req,res) => {
    await dataSource.query(
        `SELECT
                users.name,
                users.email,
                users.password
        FROM users`
        ,(err,rows)=>{
            res.status(200).json(rows);
        }
    );    
});
```